server/handlers/user_handler.coffee

```diff
@@ -497,8 +497,6 @@ UserHandler = class UserHandler extends Handler
       photoURL = document?.get('photoURL')
       if photoURL
         photoURL = "/file/#{photoURL}"
```

Comment:
I expect a very small percentage of our customers will bother setting up a Gravatar, and our existing active user avatar base is large compared to the number of people who already have one and will need to change it in the next X time period until we get rid of them.